subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/rules/VariantFilesMetadataRulesIntegrationTest.groovy

```diff
@@ -665,7 +665,7 @@ class VariantFilesMetadataRulesIntegrationTest extends AbstractModuleDependencyR
         applyFileRule | extension | not
         true          | 'jar'     | ''
         false         | 'jar'     | 'not'
-        true          | 'notJar'  | ''
+        false         | 'notJar'  | ''
         false         | 'notJar'  | 'not'
```

Comment:
@big-guy the last row of this test's data table is pretty suspicious. It passes, but I'd like your opinion on whether this is still relevant.